apps/sms/services/README.md

```diff
@@ -0,0 +1,114 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+We'll define different layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Controllers are really tied to a specific view. They'll encapsulate
```

Comment:
I just discuss an example with @julienw yesterday that: how could we define the original markThreadRead method, since it didn't map to any API directly and only used in inbox view. If we don't move these logic inside views, another possibility is moving to controller.  But not sure if this controller is only for message or all the rest of logic that in similar case(like settings).